src/Aspire.Dashboard/Components/Controls/SummaryDetailsView.razor.cs

```diff
@@ -160,6 +179,88 @@ private void SetPanelSizes(float panel1Fraction)
         _panel2Size = string.Create(CultureInfo.InvariantCulture, $"{(1 - panel1Fraction):F3}fr");
     }
 
+    public async Task OnPageKeyDownAsync(KeyboardEventArgsWithPressedKeys args)
+    {
+        if (_splitterRef is null || !args.ShiftKey)
+        {
+            return;
+        }
+
+        if (args.Key.ToLower(CultureInfo.InvariantCulture) == "t")
+        {
+            await HandleToggleOrientation();
+            return;
+        }
+
+        if (args.Key.ToLower(CultureInfo.InvariantCulture) == "o")
+        {
+            _internalShowDetails = !_internalShowDetails;
+            await InvokeAsync(StateHasChanged);
```

Comment:
Running the code I'm getting an exception when this executes after I press Shift+o on the Resources page with the details pane closed (without opening it any other way first). This seems to trigger OnParametersSet on the ResourceDetails, but the `Resource` is null. I suppose that makes sense because how would it know which one to show from the keyboard command?

Reply:
I note that if i use Shift+o to close the details, and then immediately Shift+o again to open it, it seems to work. Pressing Shift+o a third time in that scenario just types into the text area though. Slightly odd behavior for that third press, though I do understand why

Reply:
re: the exception, I needed to check that there actually is a selected resource. I'm not sure why the behavior you mention is occurring